src/app/content/components/Content.tsx

```diff
@@ -45,17 +45,18 @@ const Background = styled.div`
   }
 `;
 
+const getNotificationTop = (mobileExpanded: boolean) => mobileExpanded
```

Comment:
why not make this args `(props: {mobileExpanded: boolean })` and avoid the wrapping function on 58?